fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaLakeMetadata.java

```diff
@@ -70,6 +108,108 @@ public List<String> listPartitionNames(String databaseName, String tableName, lo
         return deltaOps.getPartitionKeys(databaseName, tableName);
     }
 
+    @Override
+    public List<RemoteFileInfo> getRemoteFileInfos(Table table, List<PartitionKey> partitionKeys,
```

Comment:
This interface is not related with expr replacing.

Reply:
This is related. In the past, files lists was obtained based on `Expr`, but now it is filtered based on `Operator`.